com.xored.f4.fantom-feature/feature.xml

```diff
@@ -1,20 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feature
       id="com.xored.f4.fantom"
-      label="Embedded Fantom"
+      label="Fantom Interpreter"
```

Comment:
As I remember We had some checks in code based on this name, I will check before merge,